src/doc/src/reference/specifying-dependencies.md

```diff
@@ -140,7 +140,8 @@ Cargo will fetch the `git` repository at this location then look for a
 of a workspace and setting `git` to the repository containing the workspace).
 
 Since we haven’t specified any other information, Cargo assumes that
-we intend to use the latest commit on the main branch to build our package.
+we intend to use the latest commit on the Default ("base") branch branch 
+to build our package, which may not necessarily be the main branch.
```

Comment:
I took the terms from GitHub's settings page. I don't actually know. I used the capital because they had it as capital, I don't feel strongly about it.IMO I think what you said is better than both. Say how it will be determined. Rather than trying to name it. As long as it is clear that it's not always main. Happy to make the edits if you prefer when I get home but you're free to edit as you see fit as well.